### ISSUE TITLE
Cartopy text in logo is off center

### ISSUE THREAD

**Issue by acertelijah** [SciTools/cartopy#1465]
### Description
I was reading the Cartopy wiki and noticed the logo. The text is not center as the 'C' in Cartopy has less left whitespace than the 'y' 

#### Code to reproduce

```
https://scitools.org.uk/cartopy/docs/latest/_static/cartopy.png
```

</details>

**Comment by philippemiron** [comment]
Hi, everyone.

I found that the origin should be set to (-171.0140625, -43.66250001) instead of the (-175, -35) currently sets in [the cartopy logo](https://scitools.org.uk/cartopy/docs/latest/gallery/logo.html).

The code to get those value is [here](https://gist.github.com/philippemiron/1da5693f721c7c9b8d76381a5330f055). 

In summary what I did is:
1. create a function that return the center of a `matplotlib.textpath.TextPath` using the same scaling as the current cartopy logo.
2. use scipy.optimize.minimize() to find the origin to horizontally centered the text 'cartopy'.
3. use scipy.optimize.minimize() to find the origin to vertically centered the letter 'o' (the letters in 'cartopy' don't extended symmetrically so scaling only with the letter 'o' in the vertical gives a better result).
